src/components/emails/index.js

```diff
@@ -50,13 +52,14 @@ class Emails extends React.Component {
   }
 
   onAddEmail() {
-    const irma = IrmaFrontend.newPopup({
+    YiviFrontend.newPopup({
       language: this.props.i18n.language,
-      session: this.props.addEmailIrmaSession,
-    });
-    irma
+      session: this.props.addEmailYiviSession,
+    })
       .start()
-      .then(() => this.props.dispatch({ type: 'startUpdateInfo' }))
+      .then(function () {
+        this.props.dispatch({ type: 'startUpdateInfo' });
+      })
```

Comment:
We use arrow functions at most other places. I think it would be nice to keep using arrow functions for consistency. Maybe we can enable https://eslint.org/docs/latest/rules/prefer-arrow-callback to enforce this style habit.

Reply:
Totally agree, noticed prefer-arrow-callback was already enabled in eslintrc, but since this function did not return a result it was not flagged by default. Tweaked its behavior so now it does:)